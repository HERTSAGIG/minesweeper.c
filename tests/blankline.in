g 8 9

b 4 5
b 2 3
b 3 3
b 9 0
b 4 3