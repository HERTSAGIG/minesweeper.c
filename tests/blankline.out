g 8 9
error
